yb-voyager/src/query/queryparser/helpers_protomsg.go

```diff
@@ -370,3 +385,21 @@ func GetSchemaAndObjectName(nameList protoreflect.List) (string, string) {
 	}
 	return schemaName, objectName
 }
+
+func TraverseAndExtractDefNamesFromDefElem(msg protoreflect.Message) []string {
+	var defNames []string
```

Comment:
just a note for future: we might need the values as well.

Reply:
added that comment in code